Src/EmbeddedScripts.JS.Jint/JintCodeRunner.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Threading.Tasks;
+using EmbeddedScripts.Shared;
+using Jint;
+
+namespace EmbeddedScripts.JS.Jint
+{
+    public class JintCodeRunner : ICodeRunner
+    {
+        public JintCodeRunner(string code)
+            : this(code, _ => JsCodeRunnerOptions.Default)
+        {
+        }
+
+        public JintCodeRunner(string code, Func<JsCodeRunnerOptions, JsCodeRunnerOptions> opts)
+        {
+            Code = code;
+
+            RunnerOptions = opts(JsCodeRunnerOptions.Default);
+        }
+
+        public JintCodeRunner WithOptions(Func<JsCodeRunnerOptions, JsCodeRunnerOptions> opts)
+        {
+            RunnerOptions = opts(JsCodeRunnerOptions.Default);
+            return this;
+        }
+
+        public JintCodeRunner AddOptions(Func<JsCodeRunnerOptions, JsCodeRunnerOptions> opts)
```

Comment:
nit: What's the use case for it?

Reply:
I've fixed that in another branch, which is already merged into this.